### PR TITLE
swagger-doc-workflow-resource-part-1

### LINKED ISSUE #29189: Workflow API swagger doc, Part 1
### Parent Issue

https://github.com/dotCMS/core/issues/28600

### Task

There's a lot of exciting work happening in the Workflow API, but it's so big that I've decided to break it up into two tasks.

This task encompasses the first ~20 endpoints in the file.

image

Screenshot 2024-07-10 at 12 54 34 PM

### Proposed Objective

Documentation

### Proposed Priority

Priority 2 - Important

### Acceptance Criteria

If Postman doesn't freak out, we should be golden~

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by jdcmsd**
### Proposed Changes
* First half of Workflow Resource has received Swagger annotations.

### Checklist
- [ ] Tests

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java [hunk 1]
```diff
@@ -271,32 +269,36 @@ public WorkflowResource() {
...
-                                                      "\n\nExample: `c541abb1-69b3-4bc5-8430-5e09e5239cc8` (Page Content Type)",
-                                              schema = @Schema(type = "string", format = "uuid",
-                                                      accessMode = Schema.AccessMode.READ_ONLY)
-                                      ) @QueryParam("contentTypeId") final String contentTypeId,
-                                      @Parameter(in = ParameterIn.QUERY, name = "showArchived",
-                                              description = "If true, includes archived schemes in response.",
-                                              schema = @Schema(type = "boolean", defaultValue = "true",
-                                                      accessMode = Schema.AccessMode.READ_ONLY)
-                                      ) @DefaultValue("true") @QueryParam("showArchived") final boolean showArchived) {
+                                      @QueryParam("contentTypeId") @Parameter(
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java:290]: nice job

**Review by jdotcms — APPROVED** [review]
it is very nice job

**Review by nollymar — APPROVED** [review]

**Comment by jdcmsd** [comment]
Seems merging trunk into branch has caused some tests to fail that weren't failing previously. Could this be just a separate trunk issue that will work itself out?

**Comment by jdcmsd** [comment]
Re-ran checks, seemed to work this time!

**Pull request merged**